shell/store/index.js

```diff
@@ -573,6 +573,18 @@ export const getters = {
     return state.targetRoute;
   },
 
+  releaseNotesUrl(state, getters) {
+    const version = getters['management/byId'](MANAGEMENT.SETTING, 'server-version')?.value;
+
+    const base = 'https://github.com/rancher/rancher/releases';
+
+    if (version && version.match(/v[0-9]+\.[0-9]+\.[0-9]$/)) {
```

Comment:
Is there anything we can get from the util? If not maybe we can extend it?

https://github.com/rancher/dashboard/blob/master/shell/utils/version.js

Reply:
Good call - I've updated the getter to use the existing function `isDevBuild`